src/components/MainBanner/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { IncrementPercentage } from '../IncrementPercentage'
+import { DataCard } from  '../DataCard'
+
+import './MainBanner.scss'
+import mexicoImage from '../../assets/mexico.jpg'
+
+export const MainBanner = () => {
+	const [country, setCountry] = useState([])
+	const API = 'https://api.unsplash.com/search/photos?client_id=rTLPnTqzq-rK3TnatKGFvmVeFGaZeYToaKUyzwcJOiA&query='
+	const exampleCountry = 'mexico'
+
+	useEffect(() => {
+		window.fetch(`${API}${exampleCountry}`)
```

Comment:
You can separate this request as a utility function on the utility folder, but  I'm not sure if its to get the background image or some country data because you are using the setCountry,  remember we have a Country singleton on utils folder, 
```
import  Countries from "../../utility/countries"

let mexico = Countries.getByAlphaCode('MX)
```
and get base information like name, Flag, set use the country state.

Reply:
I will separate the fetch used to get the **Unsplash** country random image on a utility file (and will try to make it work) and will use the countries utility to get the country name, flag and other necessary info.